### PR TITLE
bugfix: add missing comment clause to create table sql

### PR DESCRIPTION
**Pull request by beautytiger**

### CODE REVIEW

File: pkg/presto/util.go [hunk 1]
```diff
@@ -149,7 +149,7 @@ func generateCreateTableSQL(catalog, schema, tableName string, columns []Column,
 )
 %s
 %s`
-	return fmt.Sprintf(sqlStr, ifNotExists, table, columnsStr, propsStr)
+	return fmt.Sprintf(sqlStr, ifNotExists, table, columnsStr, propsStr, comment)
```
Comment (chancez) [inline pkg/presto/util.go]: comment comes before `propsStr` and we need to `prefix `COMMENT` to the message if set.

Eg:
```
commentStr := ""
if comment != "" {
  commentStr = "COMMENT " + comment
}

...
return fmt.Sprintf(sqlStr, ifNotExists, table, columnsStr, commentStr, propsStr)
```

Comment (beautytiger) [reply pkg/presto/util.go]: Got it! Thank you very much.

Comment (beautytiger) [reply pkg/presto/util.go]: Got it! Thank you very much.

File: pkg/presto/util.go [hunk 2]
```diff
@@ -136,6 +136,11 @@ func generateCreateTableSQL(catalog, schema, tableName string, columns []Column,
 
 	columnsStr := generateColumnDefinitionListSQL(columns)
 
+	commentStr := ""
+	if len(comment) != 0 {
```
Comment (chancez) [inline pkg/presto/util.go]: This can be simplified to `if comment != "" {`

**Pull request merged**